packages/ifpack2/src/Ifpack2_Relaxation_def.hpp

```diff
@@ -1837,15 +1837,15 @@ ApplyInverseGS_BlockCrsMatrix (const block_crs_matrix_type& A,
     yBlockCol->putScalar (STS::zero ());
   }
   else if (performImport) {
-    yBlockCol->doImport (yBlock, *Importer_, Tpetra::INSERT);
+    yBlockCol->getMultiVectorView().doImport (yBlock.getMultiVectorView(), *Importer_, Tpetra::INSERT);
```

Comment:
The Import object needs to be a point Import, not a block Import.

Reply:
You can get the point Map of a BlockMultiVector by calling `getPointMap()` on it.  It returns a `Map` instance.  (I know, it's not an RCP.)  I would suggest keeping a `pointImporter_` object around in the class.  Try this:
```cpp
if (pointImporter_.is_null()) {
  auto srcMap = rcp(new map_type(src.getPointMap()));
  auto tgtMap = rcp(new map_type(tgt.getPointMap()));
  pointImporter_ = rcp(new import_type(srcMap, tgtMap));
}
```